src/notebooks/controllers/commands/installPythonControllerCommands.ts

```diff
@@ -186,7 +186,7 @@ export class InstallPythonControllerCommands implements IExtensionSingleActivati
 
                     // Trigger a load of our notebook controllers, we want to await it here so that any in
                     // progress executions get passed to the suggested controller
-                    await this.controllerLoader.loadControllers(true);
+                    await this.controllerLoader.loaded;
```

Comment:
We cannot refresh controllers anymore as its lazy loaded.
`loaded` property here merely means we have loaded at least one controller, which is the same as listening to the event on this class.
Created a debt issue to resolve this (doesn't feel right to have `loaded`)